### ISSUE TITLE
[v5] Licence + Licence status displayed unclearly in the UI

### ISSUE THREAD

**Issue by tobimori** [k-next/kirby#6883]
## Description
LicenceStatus description/text doesn't show up in the UI, only in the ARIA label. 

This leads to people "abusing" the License text for showing the status, like here:
![CleanShot 2024-12-25 at 16 04 38@2x](https://github.com/user-attachments/assets/76506757-1565-4b1d-b78e-400576abd829)

And this results in inconsistency between free plugins (that show the actual License there).

![CleanShot 2024-12-25 at 16 06 09@2x](https://github.com/user-attachments/assets/ed2e477f-572b-4963-8ae3-48f5edbf3f63)

**Expected behavior**  
Both written License and LicenceStatus text should be visible at all times.

License should be MIT/GPL/etc. or "xyz Plugin License" / "Kirby EULA" / etc.

License status should be "activated", "Buy license", etc.

Consider two separate columns.

## Your setup
**Kirby Version**  5.0-beta.1

**Comment by distantnative** [comment]
I agree that two columns could make a lot of sense here, one for the license type and one for status. Some licenses (e.g. MIT) might not have any status (or always just positive).

**Comment by tobimori** [comment]
I think free plugins should have an empty activation status, because there are other CMS like Shopware that require buying a license for free plugins in the store as well, so this might be confusing for new people.

**Comment by distantnative** [comment]
Currently, trying to work on this. Would splitting it up in two columns like this make sense to you, @tobimori?

<img width="1109" alt="Screenshot 2025-01-11 at 13 50 23" src="https://github.com/user-attachments/assets/657a6b5a-5dd4-49b6-a499-34493467095f" />

Probably the strings should be a little different, e.g. as you mention maybe a free plugin shouldn't say `Valid license` then but some more neutral/passive.

**Comment by tobimori** [comment]
yes that's good!
